neo4j_utils/_driver.py

```diff
@@ -833,8 +833,7 @@ def db_status(
         query = 'SHOW DATABASES'
 
         with self.fallback():
-
-            resp, summary = self.query(query=query)
+            resp, summary = self.query(query=query, db=neo4j.DEFAULT_DATABASE)
```

Comment:
the query is executed against the `DEFAULT_DATABASE` by `query` if no other option left, isn't it? why do we have to fix it here? what if `DEFAULT_DATABASE` is not available for any reason?

Reply:
removed the assignment to the default db